src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -296,12 +304,31 @@ struct TransferCommand {
                 << (m_node.getLastLocalBlockHeaderInfo().majorVersion < CryptoNote::BLOCK_MAJOR_VERSION_4 ? m_currency.minimumFee() : m_node.getMinimalFee());
               return false;
             }
+          } 
+		  else if (arg == "-m") {
+            messages.emplace_back(value);
+          } 
+		  else if (arg == "-ttl") {
```

Comment:
What is the purpose of moving `else if (...` to a new line? Please, revert this change